frontend/src/LogsDialog.vue

```diff
@@ -40,119 +52,166 @@
 </template>
 
 <style>
-   @import 'vue-virtual-scroller/dist/vue-virtual-scroller.css';
+@import 'vue-virtual-scroller/dist/vue-virtual-scroller.css';
 
-   .ra-logs {
-      height: 80vh;
-      overflow: auto;
+.ra-logs {
+   height: 80vh;
+   overflow: auto;
 
-      background-color: black;
-      color: white;
-      font-family: var(--font-family-monospace);
+   background-color: black;
+   color: white;
+   font-family: var(--font-family-monospace);
 
-      padding: 0.5rem;
-   }
+   padding: 0.5rem;
+}
 
-   .ra-log-line {
-      white-space: nowrap;
-      height: 20px;
-   }
+.ra-log-line {
+   white-space: nowrap;
+   height: 20px;
+}
 </style>
 
 <script>
-   import Dialog from './Dialog.vue';
    import parseLinkHeader from 'parse-link-header';
-   import { DynamicScroller, DynamicScrollerItem } from 'vue-virtual-scroller'
+   import { DynamicScroller, DynamicScrollerItem } from 'vue-virtual-scroller';
+   import Dialog from './Dialog.vue';
+   import moment from 'moment';
 
    let requestUri;
+   const itemSize = 24;
 
    export default {
       data() {
          return {
             logLines: [],
-            nextPageLink: null
+            eventSource: null,
+            scrollPosition: false,
```

Comment:
Isn't following better?

```suggestion
            scrollPosition: null,
```